### PR TITLE
minor: New Projects added for Diff regression

### PR DESCRIPTION
**Pull request by Kevin222004**
minor: New Projects added for Diff regression

### CODE REVIEW

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Please explain why we don’t want to put these into one file called something like `projects-to-test-on-extended.properties`

File: checkstyle-tester/projects-to-test-on-2.properties [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+mssql-jdbc|git|https://github.com/microsoft/mssql-jdbc|main|
+OpenRefine|git|https://github.com/OpenRefine/OpenRefine|master|
+pitest|git|https://github.com/hcoles/pitest|master|
+tomcat|git|https://github.com/apache/tomcat|main|
+wildwebdeveloper|git|https://github.com/eclipse/wildwebdeveloper|master|
+mockito|git|https://github.com/mockito/mockito|main|
+processing|git|https://github.com/processing/processing|master|
+sonar-java|git|https://github.com/SonarSource/sonar-java|master|
+intellij-community|git|https://github.com/JetBrains/intellij-community|master|
+xwiki-platform|git|https://github.com/xwiki/xwiki-platform|master|
```
Comment (romani) [inline checkstyle-tester/projects-to-test-on-2.properties:27]: We have xwiki in no-error mode in CI.
Let's not use them here. Better to choose another project.

File: checkstyle-tester/projects-to-test-on-3.properties [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+weblogic-image-tool|git|https://github.com/oracle/weblogic-image-tool|main|
+react-native|git|https://github.com/facebook/react-native|main|
+weblogic-monitoring-exporter|git|https://github.com/oracle/weblogic-monitoring-exporter|main|
+nosql-java-sdk|git|https://github.com/oracle/nosql-java-sdk|main|
+java|git|https://github.com/kubernetes-client/java|master|
+spring-boot|git|https://github.com/spring-projects/spring-boot|main|
+spring-integration|git|https://github.com/spring-projects/spring-integration|main|
+spring-session|git|https://github.com/spring-projects/spring-session|main|
+spring-security|git|https://github.com/spring-projects/spring-security|main|
+RxJava|git|https://github.com/ReactiveX/RxJava|3.x|
```
Comment (romani) [inline checkstyle-tester/projects-to-test-on-3.properties:17]: https://github.com/checkstyle/contribution/blob/ce7bad08ea5d8b2c1f18ec1e425436f2fc486e2f/checkstyle-tester/projects-to-test-on.properties#L38

Let's update existing place

File: checkstyle-tester/projects-to-test-on-3.properties [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+jabref|git|https://github.com/JabRef/jabref|main|
+error-prone|git|https://github.com/google/error-prone|master|
+jpf-core|git|https://github.com/javapathfinder/jpf-core|master|
+weblogic-image-tool|git|https://github.com/oracle/weblogic-image-tool|main|
+react-native|git|https://github.com/facebook/react-native|main|
+weblogic-monitoring-exporter|git|https://github.com/oracle/weblogic-monitoring-exporter|main|
+nosql-java-sdk|git|https://github.com/oracle/nosql-java-sdk|main|
+java|git|https://github.com/kubernetes-client/java|master|
+spring-boot|git|https://github.com/spring-projects/spring-boot|main|
+spring-integration|git|https://github.com/spring-projects/spring-integration|main|
```
Comment (romani) [inline checkstyle-tester/projects-to-test-on-3.properties:14]: https://github.com/checkstyle/contribution/blob/ce7bad08ea5d8b2c1f18ec1e425436f2fc486e2f/checkstyle-tester/projects-to-test-on.properties#L54

**Review by romani — CHANGES_REQUESTED** [review]
>Please explain why we don’t want to put these into one file called something like projects-to-test-on-extended.properties

It might be too much for execution in CI. We keep config here for quick reuse.

And ... Looks like we had bad names, `projects-to-test-on.properties` is for CI jobs, where all is commented out

, but we need config for checkstyle-tester GitHub action where all lines are active.

Lets name them github-action-projects1.properties


No need for duplicate projects

**Comment by romani** [comment]
@Kevin222004 , can we finish this PR?

File: checkstyle-tester/projects-to-test-on-2.properties (hunk 1 above)
Comment (Kevin222004) [reply checkstyle-tester/projects-to-test-on-2.properties]: Done

File: checkstyle-tester/projects-to-test-on-3.properties (hunk 2 above)
Comment (Kevin222004) [reply checkstyle-tester/projects-to-test-on-3.properties]: done

File: checkstyle-tester/projects-to-test-on-3.properties (hunk 3 above)
Comment (Kevin222004) [reply checkstyle-tester/projects-to-test-on-3.properties]: done

**Comment by Kevin222004** [comment]
https://github.com/checkstyle/contribution/pull/785#issuecomment-1584782329 Sorry, for past 2 weeks I am trying to manage all the work that's why this left

**Review by romani — APPROVED** [review]
Ok to merge

File: checkstyle-tester/projects-to-test-on-for-github-action.properties [hunk 4]
```diff
@@ -32,13 +32,13 @@ apache-struts|git|https://github.com/apache/struts.git|master|
 infinispan|git|https://github.com/infinispan/infinispan|7.2.5.Final||
 protonpack|git|https://github.com/poetix/protonpack|protonpack-1.7||
 jOOL|git|https://github.com/jOOQ/jOOL|version-0.9.7||
-RxJava|git|https://github.com/ReactiveX/RxJava|v1.0.9||
+RxJava|git|https://github.com/ReactiveX/RxJava|3.x|
```
Comment (nrmancuso) [inline checkstyle-tester/projects-to-test-on-for-github-action.properties:35]: @Kevin222004 please point to where we generated report with this project

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Item:

File: checkstyle-tester/projects-to-test-on-for-github-action.properties (hunk 4 above)
Comment (Kevin222004) [reply checkstyle-tester/projects-to-test-on-for-github-action.properties:35]: @nrmancuso  https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/c6bab44_2023093734/reports/diff/index.html
this is generated today

Comment (nrmancuso) [reply checkstyle-tester/projects-to-test-on-for-github-action.properties:35]: Good, please always test/show successful execution on projects file (or any file) that we use in CI to make sure that we have not broken anything with update like this.

**Review by nrmancuso — APPROVED** [review]

**Pull request merged**